distribution/docs/src/main/resources/content/_configuring/platform-ui-theming.adoc

```diff
@@ -0,0 +1,31 @@
+:title: Configuring UI Theme
+:type: configuration
+:status: published
+:parent: Admin Theming Configurations
+:summary: Configure theme of all pages displayed on ${branding}.
+:order: 02
+
+== {title}
+
+The Platform UI Configuration allows for the customization of attributes of all pages within ${branding}.
```

Comment:
❗️ For clarity, the Platform UI Configuration is only for the Admin UI. 
I don't believe this configuration is used for Intrigue `/search`.

Reply:
Changes to the header/footer/text color/background color under platform UI config do have effects on intrigue/search under what I have tested.

Reply:
Ah okay, for some reason I thought this was explicitly different, thank you for the clarification. ✅